src/styles/icons.css

```diff
@@ -0,0 +1,54 @@
+.icon-type {
+  color: white;
+  font-weight: bold;
+  padding: 0.2rem 0.5rem;
+}
+
+.icon-family {
+  background-color: var(--colors-family-main);
+}
+
+.icon-domain {
+  background-color: var(--colors-domain-main);
+}
+
+.icon-site {
+  background-color: var(--colors-site-main);
+}
+
+.icon-repeat {
+  background-color: var(--colors-repeat-main);
+}
+
+.icon-hh {
+  background-color: var(--colors-homolog-main);
+}
+
+/*SP is that still needed ?*/
```

Comment:
What is _SP_? Single page? Swiss-Prot?

Edit: probably Seb Pesseat... Let's assume it's still needed, so please remove the comment to avoid confusions in the future.

Reply:
Yep, coming from Seb. Although, my refactoring splitting file, points to me when doing `git blame`